bin/install_openstackbmc.sh

```diff
@@ -45,12 +45,25 @@ export OS_TENANT_NAME=$os_tenant
 export OS_PASSWORD=$os_password
 export OS_AUTH_URL=$os_auth_url
 export OS_PROJECT_NAME=$os_project
-export OS_USER_DOMAIN_ID=$os__user_domain
-export OS_PROJECT_DOMAIN_ID=$os__project_domain
+export OS_USER_DOMAIN_ID=$os_user_domain
```

Comment:
Ah, sorry I should have been more explicit.  But yeah, these aren't actually variables, they're template values that get replaced.

We should probably just change the templating convention so they don't look like variables anymore, but it's going to be a little tricky since some actually are variables and others are template values. :-/